src/citrine/resources/ingestion.py

```diff
@@ -1,14 +1,16 @@
 from typing import Optional, Iterator, Iterable
 from uuid import UUID
 
+from gemd.enumeration.base_enumeration import BaseEnumeration
+
 from citrine._rest.collection import Collection
 from citrine._rest.resource import Resource
 from citrine._serialization import properties
 from citrine._session import Session
 from citrine.exceptions import CitrineException, BadRequest
 from citrine.jobs.job import JobSubmissionResponse, JobFailureError, _poll_for_job_completion
 from citrine.resources.api_error import ApiError, ValidationError
-from gemd.enumeration.base_enumeration import BaseEnumeration
+from citrine.resources.file_link import FileLink
```

Comment:
Is this introducing a circular reference risk?  I think there's a clear dependency order on these two packages, but just want to highlight the question.

Reply:
Ah, you mean moving them around? I don't know about a risk, but I know it ran fine for me locally. I moved it to adhere to PEP 8 import grouping, but if it causes problems, I'm happy to move it back.